flink-ai-flow/examples/wide_and_deep/README.md

```diff
@@ -0,0 +1,86 @@
+## Guidance on Running Wide and Deep Example
+
+### Intro
+This example, named 'wide and deep', aims at showing the mix use of [Flink-AI-Flow](https://github.com/alibaba/flink-ai-extended/blob/master/flink-ai-flow/QUICKSTART.md) and [TF-on-Flink](https://github.com/alibaba/flink-ai-extended/tree/master/deep-learning-on-flink). 
+In this example, we will train a [DNNLinearCombinedClassifier](https://arxiv.org/abs/1606.07792) for a binary classification task.
+In the codes, we will show how to train a model with TF-on-Flink, which can directly consume training samples from flink and how to define and execute the streaming training and streaming prediction tasks with AIFlow.
+
+### Environment Requirements
+1. Flink-AI-Flow([ref](https://github.com/alibaba/flink-ai-extended/blob/master/flink-ai-flow/QUICKSTART.md))
+2. TF-on-Flink([ref](https://github.com/alibaba/flink-ai-extended/tree/master/deep-learning-on-flink))
+3. Flink(1.11+)
+4. Kafka(1.x or 2.x)
+5. Hadoop
+6. Zookeeper
+
+After building TF-on-Flink and installing Flink-AI-Flow, we need to put `.jar` files under the `flink-1.11.3/lib` dir.
+Find the  `flink-ml-tensorflow-0.4-SNAPSHOT-jar-with-dependencies.jar` in your TF-on-Flink project's target dir.
+Then copy the jar and the https://repo1.maven.org/maven2/org/apache/flink/flink-sql-connector-kafka_2.11/1.11.3/flink-sql-connector-kafka_2.11-1.11.3.jar into `flink-1.11.3/lib` dir. 
+
+
+### Usage
+#### HDFS Initialization
+
+- Edit `$HADOOP_HOME/etc/hadoop/core-site.xml` of hadoop:
+    ```shell
+    <property>
+    <name>fs.default.name</name>
+    <value>hdfs://127.0.0.1:9000</value>
+    </property>
+    ```
+- Start hadoop
+    ```shell
+    hadoop namenode -format
+    $HADOOP_HOME/sbin/start-all.sh
+    ```
+- Create directory in hdfs and upload codes
+    ```shell
+    hdfs dfs -mkdir /demo
+    hdfs dfs -put ./code.zip /demo
+    ```
+  Note, the `code.zip` must be produced by zipping the `wide_and_deep/dependencies/python/code` directory.
+  If users define their own logic in the python files under this dir, they should reproduce the zip file by running
+  ```shell
+  cd dependencies/python && zip -r code.zip code
```

Comment:
It would be better if we tell user his current location before running the command